src/tailwind.css

```diff
@@ -2,19 +2,6 @@
 @source '../node_modules/@shlinkio/shlink-frontend-kit';
 @import '@shlinkio/shlink-frontend-kit/tailwind.preset.css';
 
-@utility scroll-thin {
-    /* Standard. New browsers */
-    scrollbar-width: thin;
-
-    /* Fallback */
-    &::-webkit-scrollbar {
-        width: 6px;
-    }
-    &::-webkit-scrollbar-thumb {
-        border-radius: .5rem;
-    }
-}
```

Comment:
This utility is now included in `@shlinkio/shlink-frontend-kit/tailwind.preset.css`